test/e2e/scenario/rum/resources.scenario.ts

```diff
@@ -196,6 +196,50 @@ describe('rum resources', () => {
       })
   })
 
+  createTest('track redirect fetch timings')
+    .withRum()
+    .run(async ({ serverEvents }) => {
+      await browserExecuteAsync((done) => {
+        fetch(`/redirect?duration=${200}`).then(
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
        fetch(`/redirect?duration=200`).then(
```